src/main/java/kotlinx/reflect/lite/impl/KPropertyImpl.kt

```diff
@@ -20,8 +20,6 @@ internal abstract class KPropertyImpl<out T>(
     override val isSuspend: Boolean
         get() = false
 
-    override fun call(vararg args: Any?): T = getter.call(*args)
```

Comment:
We do need this override at all, `fun call(vararg args: Any?): T` is already implemented in `KCallableImpl`